externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Conversion/TcpToArith/TcpToArith.h

```diff
@@ -0,0 +1,27 @@
+//===------------------------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TORCH_MLIR_DIALECTS_CONVERSION_TCPTOARITH_TCPTOARITH_H_
+#define TORCH_MLIR_DIALECTS_CONVERSION_TCPTOARITH_TCPTOARITH_H_
+
+#include "mlir/Pass/Pass.h"
+
+namespace mlir {
+
+#define GEN_PASS_DECL_CONVERTTCPTOARITH
+#include "torch-mlir-dialects/Conversion/Passes.h.inc"
+
+namespace tcp {
+
+std::unique_ptr<Pass> createConvertTcpToArithPass();
```

Comment:
I thought this decl was auto-generated in `torch-mlir-dialects/Conversion/Passes.h.inc`, can you please check?

Reply:
Yes, I think that is possible now. But IIRC we decided to stick to the old style for now, to keep it uniform across all conversions in TorchMLIR.

Reply:
@ramiro050 I think this is the one: https://discourse.llvm.org/t/psa-deprecation-warning-per-pass-autogenerated-macros-and-pass-options/64998